packages/utils/pack-up/packup.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
       require: './dist/cli.js',
     },
   ],
-  externals: ['node:module'],
+  externals: ['node:module', 'node:path', 'fs-extra'],
```

Comment:
how come you had to add either of these? it can handle in built modules & reads it's own `package.json` 🤔

Reply:
CLI tests were failing and this seemed to fix it, I'll try removing and see.